examples/Atlas/generateRandomBoxWorld.m

```diff
@@ -0,0 +1,129 @@
+function generateRandomBoxWorld(num_boxes, max_pose_array, max_size_array)
+%max_pose_array = [max_x, max_y] <-- goes from - max_x to + max_x
```

Comment:
If you make your second line %NOTEST then the build servers won't try to run this function as a unit test